api/app/tests/fba/test_fba_error_random_sample.py

```diff
@@ -0,0 +1,239 @@
+        # tolerances. REDapp calculates it's own ISI (doesn't take the one provided by the system),
+        # but uses a different formula from CFFDRS, so the results start getting more pronounced
+        # with higher wind speeds. For that reason we limit our wind speeds to 40m, since anything
+        # above that starts failing the unit tests.
+        wind_speed = random.uniform(0, 40)
+        wind_direction = random.uniform(0, 360)
+        temperature = random.uniform(0, 40)
+        relative_humidity = random.uniform(0, 100)
+        precipitation = random.uniform(0, 50)
+        dc = random.uniform(0, 600)
+        dmc = random.uniform(11, 205)
+        bui = bui_calc(dmc, dc)
+        ffmc = random.uniform(11, 100)
+        isi = initial_spread_index(ffmc, wind_speed)
+
+        message = (f"""({index}) elevation:{elevation} ; lat: {latitude} ; lon: {longitude}; """
+                   f"""toi: {time_of_interest}; ws: {wind_speed}; wd: {wind_direction}; """
+                   f"""temperature: {temperature}; relative_humidity: {relative_humidity}; """
+                   f"""precipitation: {precipitation}; dc: {dc}; dmc: {dmc}; bui: {bui}; """
+                   f"""ffmc: {ffmc}; isi: {isi}""")
+        logger.debug(message)
+
+        test_entry = (f"""({index}) | {fuel_type} | {elevation} | {latitude} | {longitude} | """
+                      f"""{time_of_interest} | {wind_speed} | {wind_direction} | {percentage_conifer} | """
+                      f"""{percentage_dead_balsam_fir} | {grass_cure} | {crown_base_height} | {isi}  | """
+                      f"""{bui} | {ffmc} | {dmc} | {dc} | 0.01 | 0.01 | 0.01 | 0.01 | None | 0.01 | """
+                      f"""None | 0.01 | None | 0.01 | |""")
+        logger.debug(test_entry)
+        python_input = FBACalculatorWeatherStation(elevation=elevation,
+                                                   fuel_type=fuel_type,
+                                                   time_of_interest=time_of_interest,
+                                                   percentage_conifer=percentage_conifer,
+                                                   percentage_dead_balsam_fir=percentage_dead_balsam_fir,
+                                                   grass_cure=grass_cure,
+                                                   crown_base_height=crown_base_height,
+                                                   lat=latitude,
+                                                   long=longitude,
+                                                   bui=bui,
+                                                   ffmc=ffmc,
+                                                   isi=isi,
+                                                   wind_speed=wind_speed,
+                                                   wind_direction=wind_direction,
+                                                   temperature=temperature,  # temporary fix so tests don't break
```

Comment:
I don't _think_ the "temporary fix so tests don't break" comment applies anymore?